### ISSUE TITLE
Problem when i update or reload the page

### ISSUE THREAD

**Issue by nevits8as** [makryl/jquery-asynclink#1]
The history works and change the content in the target. But when i reload the page with F5 or the button browser change the all page, with the page i call before inside the target

**Comment by makryl** [comment]
Isn't it correct behavior? How do you think it should be?

**Comment by nevits8as** [comment]
stay in the same page what i cliked before, for example, i enter in a page, and i click in the link "gallery", change the content of the div, good. But when refresh the web, the div should be stay in the gallery

**Comment by makryl** [comment]
At my example page http://aeqdev.com/tools/js/asynclink/ 
if you click "Test page", url changed to test.html, but loaded from test/test.html
then, if you press F5, browser will try to reload test.html, but not index page.

Can you show more details how do you use plugin, or give me link to your test page?

**Comment by nevits8as** [comment]
Yes i click in "test page" and the url chage to http://aeqdev.com/tools/js/asynclink/test.html but, for example, if i shared in any social page these link (http://aeqdev.com/tools/js/asynclink/test.html) the browser gives "404 Not Found" the same problem is when i press F5 in that page

**Comment by makryl** [comment]
404 error at URL http://aeqdev.com/tools/js/asynclink/test.html because I don't have full test page at server side for simplification. I have only partial pages in subdirectory ./test (for example http://aeqdev.com/tools/js/asynclink/test/test.html).
But your server should provide URLs for both full and partial pages.
